lib/notifier/index.js

```diff
@@ -0,0 +1,424 @@
+const _ = require('lodash/fp')
+
+const configManager = require('../new-config-manager')
+const logger = require('../logger')
+const machineLoader = require('../machine-loader')
+const queries = require('./queries')
+const settingsLoader = require('../new-settings-loader')
+const customers = require('../customers')
+
+const utils = require('./utils')
+const emailFuncs = require('./email')
+const smsFuncs = require('./sms')
+const { STALE, STALE_STATE, PING } = require('./codes')
+const {NOTIFICATION_TYPES: {HIGH_VALUE_TX, FIAT_BALANCE, CRYPTO_BALANCE, COMPLIANCE, ERROR}} = require('./codes')
```

Comment:
```suggestion
const {
  NOTIFICATION_TYPES: {
    HIGH_VALUE_TX,
    FIAT_BALANCE,
    CRYPTO_BALANCE,
    COMPLIANCE,
    ERROR
  }
} = require('./codes')
```

Not sure if my formatting is correct, but I guess the idea passes through